src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.inline.hpp

```diff
@@ -553,15 +547,23 @@ ShenandoahScanRemembered<RememberedSet>::verify_registration(HeapWord* address,
     // This is a mixed evacuation: rely on mark bits to identify which objects need to be properly registered
```

Comment:
Maybe this comment is too restrictive.  I think the new idea is that we use ctx != nullptr for both mixed evac and for GLOBAL collections.  Am I right about this?

Reply:
Yes, I'll update the comment to include global collections.